bot/exts/evergreen/trivia_quiz.py

```diff
@@ -231,13 +476,15 @@ async def send_score(channel: discord.TextChannel, player_data: dict) -> None:
             await channel.send("No one has made it onto the leaderboard yet.")
             return
 
-        embed = discord.Embed(colour=discord.Colour.blue())
-        embed.title = "Score Board"
-        embed.description = ""
+        embed = discord.Embed(
+            colour=discord.Colour.blue(),
```

Comment:
Lance's constants.Colours has a blue value already, you should use that for consistency with other embeds rather than using discord.py's blue

Reply:
resolved